extensions/wikia/CommunityPage/scripts/CommunityPageBenefitsModal.js

```diff
@@ -57,12 +67,36 @@ define('CommunityPageBenefitsModal',
 		}
 
 		/**
-		 * CreateComponent callback that finally shows modal
-		 * and binds submit action to Done button
-		 * One of sub-tasks for getting modal shown
+		 * CreateComponent callback that finally shows modal.
+		 * Bind tracking events.
+		 * One of sub-tasks for getting modal shown.
 		 */
 		function processInstance(modalInstance) {
 			modalInstance.show();
+
+			// Send tracking event for modal shown
+			track({
+				action: tracker.ACTIONS.IMPRESSION,
+				label: 'benefits-modal-shown'
+			});
+
+			// Bind tracking links that navigate away
+			modalInstance.$element.find('[data-track-mousedown]').bind('mousedown', function (e) {
+				track({label: $(e.target).data('trackMousedown')});
+			});
+
+			// Bind tracking clicks on elements with data-track attribute
+			modalInstance.$element.find('[data-track]').click(function (e) {
```

Comment:
Maybe we could simplify this and bind `mousedown` to all elements which need tracking?